app/controllers/api/v1/reviews_controller.rb

```diff
@@ -5,18 +5,35 @@ def create
     if user_signed_in?
       # Design descision to be made: will the react element need the wizard?
       if new_review.save
-        render json: { wizard: wizard , review: review_params }
+        render json: { wizard: wizard, review: review_params }
       else
-        render json: { wizard: wizard , review: review_params, errors: new_review.errors.full_messages }, status: :unprocessable_entity
+        render json: { wizard: wizard, review: review_params, errors: new_review.errors.full_messages }, status: :unprocessable_entity
       end
     else
-      render json: { wizard: wizard , review: review_params, errors: "Access Denied" }, status: :unprocessable_entity
+      render json: { wizard: wizard, review: review_params, errors: "Access Denied" }, status: :unprocessable_entity
+    end
+  end
+
+  def update
+
+    review = Review.find(review_params[:id])
+    if current_user.id == review_params[:user_id].to_i
```

Comment:
`review_params[:user_id]` is easily hackable - you should use the user_id that is fetched from the database to authorize the user here.